Features/Education.feature

```diff
@@ -0,0 +1,48 @@
+﻿Feature: Feature1_Education
+
+A short summary of the feature
+
+Scenario: 1 Verify user can add a new "Education" to the profile
```

Comment:
you should pass the json file through, this way you can indicate what json file you are using!

Reply:
also Im hoping there is a json file per test and not being using by other test cases